owa/app/css/openmrs-owa-idgen.css

```diff
@@ -117,21 +118,205 @@ nav{
     padding: 0 !important;
 }
 
-
-.navbar-light .navbar-nav .nav-link{
+.navbar-light .navbar-nav .nav-link {
     color: #eeeeee !important;
     font-size: 15px;
     padding-right: 20px;
     display: inline;
     list-style: none;
-
 }
 
-.nav-tabs .nav-item.show .nav-link, .nav-tabs .nav-link.active{
+.nav-tabs .nav-item.show .nav-link,
+.nav-tabs .nav-link.active {
     color: #eceeef !important;
     background-color: #009384 !important;
     border-color: #ddd #ddd #fff;
 }
 
+.clear {
+    clear: both
+}
+
+
+/* Input areas */
+
+
+/* Table styles*/
+
+.logwrapper .ReactTable {
+    margin: 20px;
+}
+
+.ReactTable .rt-resizable-header.-cursor-pointer.rt-th {
+    border-bottom: 1px solid #dddddd;
+    background: #f3f3f3;
+    width: auto;
+    text-align: left;
+    word-break: normal;
+    font-weight: bold;
+    padding: 10px 9px;
+}
+
+.rt-tr.-even {
+    background: #fffdf7;
+}
+
+.logwrapper {
+    font-family: OpenSans
+}
+
+.logwrapper .form-control,
+select.form-control {
+    border-radius: 0px;
+    line-height: 1.7em;
+}
+
+.logSearch {
+    background: #3f3d74;
+    text-align: center;
+    color: white;
+    padding: 20px 30px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -o-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -khtml-border-radius: 5px;
+    border-radius: 5px;
+    max-width: 450px;
+    margin: 0px auto 10px
+}
+
+.search_label {
+    color: #363463;
+    text-align: center;
+    line-height: 2em;
+    font-size: 1.2em;
+}
 
+.search_label span {
+    font-family: "OpenSansLight";
+    padding-left: 10px
+}
+
+input.log_searchbox {
+    margin-top: 0px;
+    font-size: 100%;
+    font-family: inherit;
+    line-height: 1.7em;
+}
+
+.searchlink {
+    background: #3f3d74;
+    color: #fffdf7;
+    border: 1px solid #fffdf7;
+    padding: 10px;
+    margin: 10px auto 0px;
+}
+
+.searchlink:hover {
+    color: #e8e4d8;
+    background: #3f3d74;
+}
+
+.advanced_wrapper fieldset>div {
+    float: left;
+    margin-bottom: 10px;
+}
+
+.advanced_wrapper fieldset input,
+select {
+    width: 100%
+}
+
+.logs_table_area {
+    position: relative;
+}
+
+.advanced_wrapper,
+.logs_table {
+    width: 100%;
+    left: 0;
+}
+
+.advanced_wrapper {
+    padding: 20px 20px 30px;
+    z-index: 10;
+    position: absolute;
+    background: rgba(255, 255, 255, 0.95);
+    border: 1px solid #3f3d74;
+    border-top-width: 5px;
+    margin-top: -15px;
+    text-align: center;
+}
+
+label.search_lbl {
+    margin-bottom: 0px
+}
+
+button {
```

Comment:
Your button selector will overide the css of other buttons, select by Id of your button